exporter/metric/metric_test.go

```diff
@@ -707,6 +707,81 @@ func (m *mock) CreateMetricDescriptor(ctx context.Context, req *monitoringpb.Cre
 	return m.createMetricDescriptor(ctx, req)
 }
 
+func TestExportWithDisableCreateMetricDescriptors(t *testing.T) {
+	for _, tc := range []struct {
+		desc                           string
+		disableCreateMetricsDescriptor bool
+		expectExportMetricDescriptor   bool
+	}{
+		{
+			desc:                           "default",
+			disableCreateMetricsDescriptor: false,
+			expectExportMetricDescriptor:   true,
+		},
+		{
+			desc:                           "Disable CreateMetricsDescriptor",
+			disableCreateMetricsDescriptor: true,
+			expectExportMetricDescriptor:   false,
+		},
+	} {
+		t.Run(tc.desc, func(t *testing.T) {
+			server := grpc.NewServer()
+
+			flag := false
```

Comment:
This could probably use a more descriptive name like `metricDescriptorCreated` or something like that

Reply:
Changed in: [33257b3](https://github.com/avilevy18/opentelemetry-operations-go/pull/1/commits/33257b31e3a9cec0824a4a00369e9f7a587ab8a8)